x-pack/plugins/maps/public/actions/store_actions.js

```diff
@@ -180,26 +187,34 @@ export function setSelectedLayer(layerId) {
   };
 }
 
-export function updateLayerOrder(newLayerOrder) {
-  return {
-    type: UPDATE_LAYER_ORDER,
-    newLayerOrder
+export function addTransientLayer(layerId, layerDescriptor) {
```

Comment:
call this `setTransientLayer`. We're not adding anything.

Reply:
Sounds good. I think I addressed this in the last comment